cockroachcloud/cmek-ops-gcp.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Provisioning Google Cloud Platform (GCP) for CMEK
+title: Provisioning GCP KMS Key and Service Account for CMEK
```

Comment:
"Provisioning KMS Keys and Service Accounts for CMEK in GCP"

"Provisioning a KMS Key and Service Account for CMEK in GCP"